bin/adam-submit

```diff
@@ -22,40 +22,15 @@ SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
 # Get list of required jars for ADAM
 ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+ADAM_CLI_JAR=$(echo "$ADAM_JARS" | tail -n 1)
 
+ADAM_JARS=$(echo "$ADAM_JARS" | tail -n +1 | perl -pe 's/\n/,/ unless eof')
```

Comment:
Er, fun times. You are correct. This should've been `head -n -1` (working too late in the night...)  but I swapped that because BSD `head` doesn't support the `-n -1` flag. Let me find a good, portable fix and I'll ping back.